experimental/i18n-dev.crontab

```diff
@@ -15,21 +15,21 @@
 #
 # Where can I ask a question about this config?
 # Reach out to the #i18n slack channel.
-
+PATH=/home/ubuntu/.rbenv/shims:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

Comment:
Yeah, this is what was previously being done. It works, my only thing is from a discovery standpoint, it's a bit tricky that this PATH will override the one being set in the `~/.bashrc` file, which is what I was trying to get cron to pick up. But this works, so I say we just do it this way and be done with it.

Reply:
One thing I noticed was that `~/.bashrc` *didn't* include a $PATH update .. so while I understand that from a discovery point of view, `~/.bashrc` is where $PATH updates should be, I didn't want to update the whole bashrc for everyone :/

Reply:
Oh, one more thing. When I `echo $PATH`, nvm has somehow got loaded in, but I don't see it being added here in the hardcoded PATH. I think we need to add: `/home/ubuntu/.nvm/versions/node/v18.16.0/bin` to the PATH variable here.